developer_guides/debugability/probes/index.rst

```diff
@@ -3,49 +3,54 @@
 Probes
 ######
 
-Typically pipeline for audio data processing contains several components separated by data buffers,
-probe module is a debug feature which allows to extract or inject data from these buffers.
-It may help in finding audio issues or bugs in audio components with the possibility to
-analyze data from each buffer separately. 
+Typically, pipeline for audio data processing contains several components
+separated by data buffers; the probe module is a debug feature that allows
+for data extraction from (or injection into) these buffers. It aids in
+finding audio issues or bugs in audio components with possible data analysis
+from each buffer.
 
 Requirements
 ************
 
-* install `tinycompress <https://github.com/alsa-project/tinycompress>`_ (crecord tool)
-               
+- Install `tinycompress <https://github.com/alsa-project/tinycompress>`_ (crecord tool)
+
 Enabling Probes
 ***************
 
-* enable Linux kernel configuration option: DEBUG_FS
-* enable Probe module in SOF firmware kconfig using command:
+- Enable the Linux kernel configuration option: ``DEBUG_FS``
+- Enable the Probe module in the SOF firmware kconfig using this command:
 
 	.. code-block:: bash
 
 		make menuconfig
 
-* :ref:`build firmware <build-from-scratch>`
+- :ref:`Build the firmware <build-from-scratch>`
 
-Note: there is no need to modify audio topology file
+Note that you do not need to modify the audio topology file.
 
 Data extraction
 ***************
 
-Extraction is the most common use case, it allows to extract data from audio component data buffer.
-It requires to start compress stream by crecord tool, one compress stream may contain
-data from several extraction probe points which means data parsing is needed at the last stage of extraction.
+Extraction is the most common use case. It allows for data extraction from
+the audio component data buffer. It requires starting the compress stream by
+starting the crecord tool. One compress stream may contain data from several
+extraction probe points which means data parsing is needed at the last stage
+of extraction.
+
+- Start the crecord tool to prepare the extraction stream (read the crecord
+  readme file)
+- Use ``aplay`` to start the playback stream
+- (optionally) Pause the playback stream
+- Add probe points via the ``debugfs`` "probe_points" entry in ``/sys/kernel/debug/sof``
 
-* Start the crecord tool to prepare extraction stream (read crecord readme)
-* Use aplay to start playback stream
-* (optionally) Pause playback stream
-* Add probe point via debugfs "probe_points" entry in "/sys/kernel/debug/sof"
-	for example to add buffer 7 probe point:
+  For example, to add a buffer with 7 probe points:
```

Comment:
@deb-intel  Sorry for being late (no notifications from this PR) but this change will mislead the user.
it should be 'to add a probe point with buffer id=7' or something like that, single probe point